plugins/inputs/cisco_telemetry_gnmi/README.md

```diff
@@ -0,0 +1,61 @@
+# Cisco GNMI telemetry
+
+Cisco GNMI telemetry is an input plugin that consumes telemetry data similar to the [GNMI specification](https://github.com/openconfig/reference/blob/master/rpc/gnmi/gnmi-specification.md).
+This GRPC-based protocol can utilize TLS for authentication and encryption.
+
+This plugin has been developed to support GNMI telemetry as produced by Cisco IOS XR (64-bit) version 6.5.1 and later.
+
+
+### Configuration:
+
+This is a sample configuration for the plugin.
+
+```toml
+[[inputs.cisco_telemetry_gnmi]]
+  ## Address and port of the GNMI GRPC server
+  address = "10.49.234.114:57777"
```

Comment:
You can already add multiple instances of the plugin pointing to different devices.

Reply:
While multiple plugins can be used, it could be nice from a configuration point of view to be able to list multiple devices to reduce the duplication.

One downside of connecting to multiple devices in a single plugin instance is that if one device blocks the input then it can prevent other devices from reporting, though this is mostly an issue with polling plugins.  For this plugins in particular the main downside would be more complexity managing multiple connections.

We should consider how many devices one may want to monitor, especially with the same set of subscriptions.

Reply:
Okay, I added it in https://github.com/influxdata/telegraf/pull/5852/commits/eb13faa4a7ad8f55b51712c7d556aae5f55ae7bb it wasn't that complex.